create-vm.yml

```diff
@@ -0,0 +1,84 @@
+---
+# This playbook creates VM from the QCOW2 Image
+
+- name: Create VM Playbook
+  hosts: localhost
+  become: yes
+  become_method: sudo
+
+  vars_files:
+    - vars/guests.yml
+
+  tasks:
+    - name: Start libvirtd
+      service: name=libvirtd state=started enabled=yes
+      register: libvirtd
+
+    - name: Wait for libvirtd to start
+      wait_for:
+        timeout: 30
+      when: libvirtd.changed
+
+    - name: Create domain {{ domain }}
+      command:
+        echo 'Empty message'
```

Comment:
I wanted to display only the message `Create domain {{ domain }}`. Not sure how to do **nothing** in the play. Just like we do `pass` in Python.